.github/workflows/changelog.yml

```
@@ -0,0 +1,17 @@
+name: Changelog
+on: [pull_request]
+
+jobs:
+  fragments:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Fetch Go version from .go-version
+      run: echo "GO_VERSION=$(cat .go-version)" >> $GITHUB_ENV
+    - uses: actions/setup-go@v3
+      with:
+        go-version: ${{ env.GO_VERSION }}
+    - name: check changelog
```

NIT: Can we make this step name more meaningful?

---

what name do you have in mind?

---

I'd assume `pr-has-fragment` provides more meaning 
```suggestion
    - name: check pr-has-fragment
```